### ISSUE TITLE
TLS not working

### ISSUE THREAD

**Issue by vollkommenIrrelevant** [osTicket/osTicket-plugins#210]
![grafik](https://user-images.githubusercontent.com/25504254/129542268-03df83b7-56b9-45e0-9f32-55a171d5ed09.png)

`openssl s_client -debug -connect $DOMAIN_CONTROLLER:389 -starttls ldap -showcerts` 
Is showing that ssl is working on side of Domain like a charme.

**Comment by vollkommenIrrelevant** [comment]
Ps. It would be great if the Error response will be add to the error message. So you can see what is going wrong...

**Comment by vollkommenIrrelevant** [comment]
bump

**Comment by bob4os** [comment]
Is your domain controller using a self-signed certificate?
If so, did you import your CA root certificate?

**Comment by vollkommenIrrelevant** [comment]
> Is your domain controller using a self-signed certificate? If so, did you import your CA root certificate?

yes and no, for shure it is the solution. But it make no sense to search the error if you didn't get more information with the error message.
